satellite/satellitedb/users.go

```diff
@@ -56,6 +56,7 @@ func (users *users) Insert(ctx context.Context, user *console.User) (_ *console.
 		dbx.User_PasswordHash(user.PasswordHash),
 		dbx.User_Create_Fields{
 			ShortName: dbx.User_ShortName(user.ShortName),
+			PartnerId: dbx.User_PartnerId(user.PartnerID[:]),
```

Comment:
what happens if PartnerID is not set?

Reply:
PartnerID column is defined as a filed nullable.